Mobile/starter_project_dartsmiths/lib/core/utils/article_page_styles.dart

```diff
@@ -0,0 +1,38 @@
+import 'package:flutter/material.dart';
+
+
+class BlogFontConstant {

```

Comment:
this shouldn't be a class the constants should exist without being an attribute to a class